extensions/dataseeding/catalog/src/main/resources/nodes-local.json

```diff
@@ -12,12 +12,5 @@
     "supportedProtocols": [
       "ids-multipart"
     ]
-  },
-  {
-    "name": "connector3",
```

Comment:
Is this change expected?
Seems weird

Reply:
typically, on a local environment we only have two connectors running, so it would just add log noise.